### PR TITLE
Throwing tweaks

### PR DESCRIPTION
**Pull request by metalgearsloth**
Landing is when the timer stops, throwing stops when it stops moving.
@SweptWasTaken @mirrorcult 

Fixes https://github.com/space-wizards/space-station-14/issues/4654

:cl:
- fix: Items can be thrown into disposals more easily again.

### CODE REVIEW

File: Content.Shared/Throwing/LandEvent.cs [hunk 1]
```diff
@@ -5,31 +5,13 @@
...
-        ///     Location where the item landed.
-        /// </summary>
-        public EntityCoordinates LandLocation { get; }
-
-        public LandEvent(IEntity? user, IEntity thrown, EntityCoordinates landLocation)
-        {
-            User = user;
-            Thrown = thrown;
-            LandLocation = landLocation;
-        }
```
Comment (mirrorcult) [inline Content.Shared/Throwing/LandEvent.cs:33]: was this all just removed because nothing used it?

**Review by mirrorcult — APPROVED** [review]

**Comment by mirrorcult** [comment]
seems like best of both worlds to me

File: Content.Shared/Throwing/LandEvent.cs (hunk 1 above)
Comment (Zumorica) [reply Content.Shared/Throwing/LandEvent.cs:33]: At the very least, keep the `User` one imo

Comment (metalgearsloth) [reply Content.Shared/Throwing/LandEvent.cs:33]: Okey dokey

**Pull request merged**